src/dotnet/App.Maui/Services/ClientAppSettings.cs

```diff
@@ -16,6 +23,9 @@ public record ClientAppSettings
         }
     }
 
+    public Uri BaseUri { get; }
+    public string BaseUrl { get; }
```

Comment:
Is it important to have both? isn't BaseUri sufficient?

Reply:
They both are used. I would keep both.